src/helm/proxy/server.py

```diff
@@ -84,6 +84,14 @@ def perform(args):
     return safe_call(perform)
 
 
+@app.get("/api/tokenizer_info")
+def handle_get_tokenizer_info():
```

Comment:
Rename to `/api/get_window_service_info` and `handle_get_window_service_info`. See below for reasoning.